trunk8.js

```diff
@@ -8,7 +8,15 @@
  * Date: September 26, 2012
  */
 
-(function ($) {
+(function (root, factory) {
```

Comment:
I'm not familiar with this pattern. Do you have a link to some docs describing what this does/is for?